src/sensor/ping360asciiprotocol.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include <QObject>
+
+/**
+ * @brief Helper struct for discovery message response
+ * It is defined by the format:
+ * Sensor Name\\r\\n
+ * Organization\\r\\n
+ * MAC Address:- 00-00-00-00-00-00\\r\\n
+ * IP address:- 127.000.000.001\\r\\n
+ *
+ * E.g: "SONAR PING360  \r\nBlue Robotics\r\nMAC Address:- 54-10-EC-79-7D-C5\r\nIP Address:- 192.168.000.111\r\n"
+ *
+ */
+struct Ping360DiscoveryResponse {
+    QString deviceName;
+    QString organization;
+    QString macAddress;
+    QString ipAddress;
+
+    static const int numberOfLines = 5;
+};
+
+/**
+ * @brief Helper class that provides an abstraction over the human friendly protocol of Ping360
+ * This is implemented following the "Ping360 Sonar Blue Robotics Communications Protocol v2.0" PDF.
+ *
+ */
+class Ping360AsciiProtocol : public QObject {
+    Q_OBJECT
+public:
+    /**
+     * @brief The Discovery message used by the host computer to find Ping360.
+     *
+     * @return QByteArray
+     */
+    static QByteArray discoveryMessage();
```

Comment:
const static variable instead of static method?

Reply:
In pingviewer we usually do this kind of thing in structures, since they are data encapsulation. 
Ping360AsciiProtocol, as a class, uses functions, there is some methods that require arguments, and having messages that are not arguments but yes variables, will result in a confuse API, we should be consistent.